js_modules/dagster-ui/packages/ui-core/src/workspace/CodeLocationRowSet.tsx

```diff
@@ -117,7 +117,7 @@ export const LocationStatus = (props: {
     );
   }
 
-  if (locationOrError?.updatedTimestamp !== locationStatus.updateTimestamp) {
+  if (locationOrError?.versionKey !== locationStatus.versionKey) {
```

Comment:
This will work in cloud too?

Reply:
yea, in the `CodeLocation` subclass there will effectively use `updateTimestamp` since thats appropriate there, where we use code server id here in oss deployments